src/hoprd_deployment.rs

```diff
@@ -92,6 +92,8 @@ pub async fn build_deployment_spec(labels: BTreeMap<String, String>, hoprd_spec:
                         image_pull_policy: Some("Always".to_owned()),
                         ports: Some(build_ports(p2p_port).await),
                         env: Some(build_env_vars(&hoprd_spec, &hoprd_secret, p2p_port)),
+                        // command: Some(vec!["/bin/bash".to_owned(), "-c".to_owned()]),
```

Comment:
Yes, I added these commented lines in case I need to debug the deployment container. I used to test the TCP and UDP connections
Not intended to be enabled when merging never.